src/traffic_server/InkAPI.cc

```diff
@@ -8962,6 +8962,17 @@ TSVConnSSLConnectionGet(TSVConn sslp)
   return ssl;
 }
 
+tsapi TSSslVerifyCTX
+TSVConnSslVerifyCTXGet(TSVConn sslp)
+{
```

Comment:
A confusion aspect of the TS API is that some of the API function implementations do null pointer and other sanity checking on parameters and others don't.

Reply:
The API should be doing some basic argument testing IMO.